pluribus/poker/dealer.py

```diff
@@ -7,41 +7,50 @@
 if TYPE_CHECKING:
     from pluribus.poker.table import PokerTable
     from pluribus.poker.player import Player
+    from pluribus.poker.card import Card
 
 
 class Dealer:
     """The dealer is in charge of handling the cards on a poker table."""
 
-    def __init__(self):
-        self.deck = Deck()
-        self.deck.shuffle()
+    def __init__(self, **deck_kwargs):
+        self.deck = Deck(**deck_kwargs)
 
-    def deal_card(self):
-        return self.deck.pick()
-
-    def use_fresh_deck(self):
-        self.deck = Deck()
-        self.deck.shuffle()
+    def deal_card(self) -> Card:
```

Comment:
Ok, great so it looks like all we had to do was allow for a random card from the deck rather than the next card.